### ISSUE TITLE
reset hiddenautoinc

### ISSUE THREAD

**Issue by domenicoo** [splitbrain/dokuwiki-plugin-bureaucracy#149]
Hi,

I could not find in the documentation how to reset the counter in hiddenautoinc.
Any pointers?

Thank you,
Domenico

**Comment by Klap-in** [comment]
What happens if you provide a default value (e.g. `=45`)?
If that doesn't work, does it maybe work if you change the label?

**Comment by domenicoo** [comment]
> does it maybe work if you change the label?

yes, this did the trick (changed the label and then changed it back).

Thanks a lot,
Domenico

**Comment by Floffyko** [comment]
When a form has more dependencies on more pages, this solution can bring damages, reseting all others instances or pages where will be use, in my case in conjunction with pagemode, plugin. 

The hidden counter is brilliant but we need a low level control. Let's start by finding where store this function the last entry in order to be able to modify it.  In my case it is sure, it does not read it from a page like `number ++`, I assume.

**Comment by virk** [comment]
Is it known by somebody where these auto-increment ++-values are stored?

**Comment by martingabelmann** [comment]
From ``lib/plugins/bureaucracy/helper/fieldnumber.php`` I learned that the counter is stored in ``data/meta/pagename.meta`` of the corresponding page where the form is located. There search for ``<field label>_autoinc";i:<num>;`` where ``<field label>`` is the label of the autoinc field and `<num>` is the counter of the next iteration (i.e. already incremented).  
Changing `<num>` to a higher value worked for me (not quite the same as a reset but also related...).
